src/common/services/Solana/GetData/ReadTokenWallet.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  clusterApiUrl,
+  Connection,
+  PublicKey,
+  ParsedAccountData,
+} from '@solana/web3.js';
+
+const TOKEN_PROGRAM_ID = new PublicKey(
+  'TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA'
+);
+
+type ItokenOwned = {
+  mint: string;
+  uiAmount: number;
+};
+
+export default async function ReadTokenWallet(
+  publickey: string
+): Promise<ItokenOwned[]> {
+  const connection = new Connection(clusterApiUrl('devnet'), 'confirmed');
```

Comment:
Use the same config file I mentioned above instead of the hardcoded 'devnet'

Reply:
sure